src/ui/facet.js

```diff
@@ -93,11 +93,35 @@ define(function(require, exports, module) {
         .value()
         .join('');
       this.$node.html(
+// <<<<<<< HEAD
         facet +
         templates.facetControls({
           showResults: this.showAllFacets,
           facetOffset: this.facetOffset + 1
         })
+// =======
+        // _.chain(facetData)
+        //   .map(
+        //     _.bind(function(values, key) {
+        //       var has_selected = _.some(values, 'selected');
+        //       // render a template for each facet
+        //       return templates.facet({
+        //         title: this.facetConfig[key].title,
+        //         key: key,
+        //         // render the form for each value of the facet
+        //         form: templates.form({
+        //           key: key,
+        //           has_selected: has_selected,
+        //           inputs: _.chain(values)
+        //             .filter('count')
+        //             .map(templates.input)
+        //             .value()
+        //         })
+        //       });
+        //     }, this))
+        //   .value()
+        //   .join('')
+// >>>>>>> upstream/gh-pages
```

Comment:
need help here deciphering how to resolve this conflict. HEAD works and keeps the questionaire in tact, but the upstream changes breaks it.